docs/docs/developers/tutorials/codealong/first_fees.md

```diff
@@ -0,0 +1,354 @@
+  await waitForPXE(pxe);
+  // use pxe...
+}
+```
+
+## Create Account Contract in a PXE
+
+**TODO: as snippet**
+
+An account on Aztec is a smart contract that specifies a method of authentication and a method of payment, allowing it to be used by the protocol to perform a transaction.
+
+For convenience, Aztec has implemented an account contract that authenticates transactions using Schnorr signatures. The contract class for a Schnorr account is pre-registered on the network to bootstrap first use. Ordinarily for a contract to be deployed, its class would have to be register with the network first.
+
+When a PXE creates a Schnorr account, there are three key things that occur:
+ - generation of keys privately
+ - registeration of a Schnorr account locally (effectively an instance of the Schnorr account class)
+   - at this stage the address of the account is known, even though it is not yet deployed
+ - initialization and deployment of the account contract to the network
+
+**Create the keys and account in the PXE**
+
+Lets first create the account but only register the instance in the pxe. Its address will be calculated, and in a later step we can deploy it counterfactually to the network.
+
+```bash
+aztec-wallet create-account --register-only -a main
+```
+
+The `-a main` sets this new account's alias to, "main". Use `aztec-wallet get-alias` (with no params) to see all aliases.
+
+The equivalent using aztec.js - create a random account locally (we will deploy it in the next step):
+
+```javascript
+import { Fr } from "@aztec/aztec.js";
+import { getSchnorrAccount } from '@aztec/accounts/schnorr';
+async function main() {
+  let secretKey = Fr.random(); let salt = Fr.random();
+  let schnorrAccount = await getSchnorrAccount(pxe, secretKey, deriveSigningKey(secretKey), salt);
+}
+```
+
+Your PXE now has keys for an account that can be deployed on Aztec network(s).
+
+## Paying for an account deployment transaction
+
+To make transactions on the network, your account contract will need to specify a payment method of the enshrined asset, "Fee Juice".
+
+### Sponsored FPC
+
+To bootstrap first use, a sponsored fee paying contract (the canonical sponsored FPC) can be used to deploy your first account.
+
+**TODO: as snippet**
+A fee paying contract (FPC) effectively implements fee abstraction. It is a contract that pays for transactions of other accounts, when its own custom criteria is met.
+In the case of the canonical sponsored FPC, the only criteria is an upper bound on how much it sponsors an account's transactions. This will be enough to at least deploy an account.
+
+The PXE can be queried for the canonical sponsored FPC address, and then specified as the payment method.
+
+Via the CLI:
+
+The alias set earlier can be confirmed using: `aztec-wallet get-alias accounts:main`, this is specified here in `--from main`.
+
+```bash
+SPONSORED_FPC_ADDRESS=`aztec-wallet --get-canonical-sponsored-fpc-address`
```

Comment:
```suggestion
aztec-wallet register-contract contracts:sponsoredfpc SponsoredFPC --from main
```
The address is at alias `contracts:sponsoredfpc`. You need to register it in the `main` account PXE before you can use it

Reply:
The alias was not set for me, but updated.